backend/src/main/java/hu/flowacademy/worksheet/controller/PartnerController.java

```diff
@@ -66,10 +67,11 @@ public Partner getPartnerById(@PathVariable("id") String id) throws ValidationEx
         return partnerService.getPartnerById(id);
     }
 
-    @PutMapping("/partners/{id}/setstatus")
-    @RolesAllowed({"admin"})
-    public Partner setUserStatus(@PathVariable("id") String id) throws ValidationException {
-        return partnerService.togglePartnerActivity(id);
+    @PutMapping("/partners/{id}/{status}")
```

Comment:
Miért került le innen a szerepkör vizsgálat?

Reply:
javítottam.

Reply:
Jó ez így, én azon csodálkozok, hogy eddig ott volt. Business szerint Partnert admin és munkavállaló is szerkeszthet